### PR TITLE
Methods only called by inner classes should move to those classes

### PR DESCRIPTION
**Pull request by Jonkno**
Rule:  "private" methods called only by inner classes should be moved to those classes

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/UploadLayout.java [hunk 1]
```diff
@@ -215,13 +215,18 @@ private void processFile(final Html5File file) {
...
-    private static boolean isDirectory(final Html5File file) {
-        if (Strings.isNullOrEmpty(file.getType()) && file.getFileSize() % 4096 == 0) {
-            return true;
+        private StreamVariable createStreamVariable(final Html5File file) {
+            return new UploadHandler(file.getFileName(), file.getFileSize(), UploadLayout.this, uploadInfoWindow,
+                    spInfo.getMaxArtifactFileSize(), null, file.getType());
+        }
+
+        private boolean isDirectory(final Html5File file) {
+            if (Strings.isNullOrEmpty(file.getType()) && file.getFileSize() % 4096 == 0) {
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/UploadLayout.java]: return the if statement directly

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 2]
```diff
@@ -267,6 +298,43 @@ private void doAssignments() {
...
+            for (final TagData tagData : tokensSelected.values()) {
+                if (tagManagement.findTargetTagById(tagData.getId()) == null) {
+                    deletedTags.add(tagData.getName());
+                } else {
+                    targetManagement.toggleTagAssignment(
+                            managementUIState.getTargetTableFilters().getBulkUpload().getTargetsCreated(),
+                            tagData.getName());
+                }
+            }
+            if (!deletedTags.isEmpty()) {
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: early return

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgroup/RolloutGroupListGrid.java [hunk 3]
```diff
@@ -391,6 +381,18 @@ public String convertToPresentation(final RolloutGroupStatus value, final Class<
         public Class<String> getPresentationType() {
             return String.class;
         }
+        
+        private String convertRolloutGroupStatusToString(final RolloutGroupStatus value) {
+            StatusFontIcon statusFontIcon = statusIconMap.get(value);
+            if (statusFontIcon == null) {
+                return null;
+            }
+            String codePoint = statusFontIcon.getFontIcon() != null
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgroup/RolloutGroupListGrid.java]: Duplicate code with RolloutGroupTargetsListGrid.java row 240

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgrouptargets/RolloutGroupTargetsListGrid.java [hunk 4]
```diff
@@ -238,16 +236,27 @@ public String convertToPresentation(final Status status, final Class<? extends S
...
+        private String getStatus() {
+            final RolloutGroup rolloutGroup = rolloutUIState.getRolloutGroup().isPresent()
+                    ? rolloutUIState.getRolloutGroup().get() : null;
+            if (rolloutGroup != null && rolloutGroup.getStatus() == RolloutGroupStatus.READY) {
+                return HawkbitCommonUtil.getStatusLabelDetailsInString(
+                        Integer.toString(FontAwesome.DOT_CIRCLE_O.getCodepoint()), "statusIconLightBlue", null);
+            } else if (rolloutGroup != null && rolloutGroup.getStatus() == RolloutGroupStatus.FINISHED) {
+                return HawkbitCommonUtil.getStatusLabelDetailsInString(
+                        Integer.toString(FontAwesome.MINUS_CIRCLE.getCodepoint()), "statusIconBlue", null);
+            } else {
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgrouptargets/RolloutGroupTargetsListGrid.java]: else is not necessary

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 5]
```diff
@@ -267,6 +298,43 @@ private void doAssignments() {
...
+                            managementUIState.getTargetTableFilters().getBulkUpload().getTargetsCreated(),
+                            tagData.getName());
+                }
+            }
+            if (deletedTags.isEmpty()) {
+                return null;
+            }
+            if (deletedTags.size() == 1) {
+                return i18n.get("message.bulk.upload.tag.assignment.failed", deletedTags.get(0));
+            } else {
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: else is not necessary

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 6]
```diff
@@ -267,6 +298,43 @@ private void doAssignments() {
...
 
+        private String saveAllAssignments() {
+            final ActionType actionType = ActionType.FORCED;
+            final long forcedTimeStamp = new Date().getTime();
+            final TargetBulkUpload targetBulkUpload = managementUIState.getTargetTableFilters().getBulkUpload();
+            final List<String> targetsList = targetBulkUpload.getTargetsCreated();
+            final DistributionSetIdName dsSelected = (DistributionSetIdName) comboBox.getValue();
+            if (distributionSetManagement.findDistributionSetById(dsSelected.getId()) == null) {
+                return i18n.get("message.bulk.upload.assignment.failed");
+            } else {
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: else is not necessary

**Comment by michahirsch** [comment]
@Jonkno please merge with master to be able to merge it.

**Comment by Jonkno** [comment]
@michahirsch doing it at the moment

**Pull request merged**